example/prepare_docker_test_data.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
See comment in action yaml, may be worth turning this into a full test and compare, delete script that can be executed in the container during build.

Reply:
As above